### PR TITLE
DYN-5985 XmlDocumentationsUtility Migration

### PR DESCRIPTION
**Pull request by GuillermoRCADSK**
XmlDocumentationsUtility project added to .Net6 Solution

Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

XmlDocumentationsUtility project added to .Net6 Solution

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes

XmlDocumentationsUtility project added to .Net6 Solution

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/Dynamo.All.Net6.sln [hunk 1]
```diff
@@ -330,6 +331,24 @@ Global
 		{F382C3F8-C55C-4350-800A-3D13A94F8E13}.Debug|NET60_Windows.ActiveCfg = Debug|NET60_Windows
 		{F382C3F8-C55C-4350-800A-3D13A94F8E13}.Debug|NET60_Windows.Build.0 = Debug|NET60_Windows
 		{F382C3F8-C55C-4350-800A-3D13A94F8E13}.Release|NET60_Windows.ActiveCfg = Release|NET60_Windows
+		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|x64.ActiveCfg = Release|x64
+		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|x64.Build.0 = Release|x64
+		{4CAB9005-0001-4B03-BBA0-57070ED105F1}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (mjkkirschner) [inline src/Dynamo.All.Net6.sln:336]: Please remove these other platforms from this sln, we only need `NET60_Windows` and `NET60_Linux`.
I think if you base your work on master first they will already be gone. @pinzart90 recently removed them I think.

Comment (pinzart90) [reply src/Dynamo.All.Net6.sln:336]: Only windows for all.net6
We do not support Linux for our UI

Comment (mjkkirschner) [reply src/Dynamo.All.Net6.sln:336]: yes, thanks for the correcttion @pinzart90 - though I think before @GuillermoRCADSK does more work here, lets decide if we can just delete this old tool.

Comment (GuillermoRCADSK) [reply src/Dynamo.All.Net6.sln]: Done

**Comment by GuillermoRCADSK** [comment]
XmlDocumentationsUtility project removed @mjkkirschner

**Comment by sm6srw** [comment]
Thus PR is now empty? Should it be closed?

**Comment by mjkkirschner** [comment]
No, it seems @GuillermoRCADSK misunderstood - don't just remove the project from the sln - remove the project and its source code.

**Pull request closed without merging**